rama-net/src/client/mod.rs

```diff
@@ -2,7 +2,7 @@
 
 mod conn;
 #[doc(inline)]
-pub use conn::{ConnectorService, EstablishedClientConnection};
+pub use conn::{ConnectorService, EstablishedClientConnection, MockConnectorService};
```

Comment:
again I would expose this from within a separate module which makes it clear that it's a test utility